R/html_document.R

```diff
@@ -287,6 +293,16 @@ html_document <- function(toc = FALSE,
         base64enc::base64encode(input),
         '</div>')
     }
+    if (!is.null(proj_download)) {
+      proj_zip_file <- file.path(tempdir(), paste0(basename(proj_download), ".zip"))
+      zip(zipfile = proj_zip_file, flags = "-r9Xq", files = list.files(proj_download, recursive = TRUE, no.. = TRUE))
+      proj_zip <<- paste0(
+        '<div id="proj-zip">',
+        base64enc::base64encode(readBin(proj_zip_file, "raw",
```

Comment:
FWIW I think the `base64enc` function also accepts file paths directly (so you don't need to read into R, and then encode)